### PR TITLE
edits 032122

### PR DESCRIPTION
**Pull request by lizz7000**
edits for your review, per the VMware Marketing Editorial Style Guide.

### CODE REVIEW

File: content/guides/kubernetes-mtls/index.md [hunk 1]
```diff
@@ -221,26 +227,28 @@ spec:
...
-can be queried, which specifically interacts with Redis:
+For extra verification, the client application comes with an API endpoint that can be queried. 
+This specifically interacts with Redis:
 
 ```shell
 kubectl port-forward svc/spring-boot-redis-client-app 8080:8080 -n mtls-demo
 
 # In another terminal window/tab
-curl -XGET http://localhost:8080/
+curl -XGET http://localhost:8080/.
```
Comment (ddubson) [inline content/guides/kubernetes-mtls/index.md]: **[nit]**: trailing dot may be misleading to the reader attempting to `curl` this address, if I'm not mistaken.

File: content/guides/kubernetes-mtls/index.md [hunk 2]
```diff
@@ -643,32 +653,34 @@ spec:
...
+must be the same as the contents of the Secret `redis-client-certificate-keystore-password`.
 
 Verify that the application has now connected to a TLS-enabled Redis server:
 
 ```shell
 kubectl port-forward svc/spring-boot-redis-client-app 8080:8080 -n mtls-demo
 
 # In another terminal window/tab
-curl -XGET http://localhost:8080/
+curl -XGET http://localhost:8080/.
```
Comment (ddubson) [inline content/guides/kubernetes-mtls/index.md]: **[nit]**: same issue as above comment around `curl`-ing issues with a trailing period.

File: content/guides/kubernetes-mtls/index.md [hunk 3]
```diff
@@ -837,28 +849,27 @@ kubectl logs deploy/spring-boot-redis-client-app -n mtls-demo | head -n1
...
```
Comment (ddubson) [inline content/guides/kubernetes-mtls/index.md]: **[nit]**: same error as mentioned above.

**Review by ddubson — CHANGES_REQUESTED** [review]
Thank you so much for taking the time to review and edit my post. Greatly appreciate it! I left a few minor nitpick comments, but otherwise, I 👍🏼  the changes.

File: content/guides/kubernetes-mtls/index.md (hunk 1 above)
Comment (lizz7000) [reply content/guides/kubernetes-mtls/index.md]: Fixed.

File: content/guides/kubernetes-mtls/index.md (hunk 2 above)
Comment (lizz7000) [reply content/guides/kubernetes-mtls/index.md]: fixed.

File: content/guides/kubernetes-mtls/index.md (hunk 3 above)
Comment (lizz7000) [reply content/guides/kubernetes-mtls/index.md]: fixed. :-)

**Pull request merged**